src/nidigital/metadata/functions.py

```diff
@@ -1971,6 +1992,7 @@
             },
             {
                 'direction': 'in',
+                'python_api_converter_name': 'convert_site_string',
```

Comment:
I think we should use repeated capability here. `convert_site_string` should be restricted HRAM methods.

Reply:
The reason I changed to use the converter is because the name and the method name implies a single site, not a list of sites.

Given this, do you still want me to change to a repeated capability?

Reply:
Fair point about the method name, I'm ok to leave it as converter.